org.wso2.codequalitymatrices/src/main/java/com/wso2/code/quality/matrices/PmtApiCaller.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.code.quality.matrices;
+
+import org.apache.http.client.methods.HttpGet;
+
+import static com.wso2.code.quality.matrices.model.Constants.AUTHORIZATION;
+import static com.wso2.code.quality.matrices.model.Constants.BEARER;
+
+/**
+ * This is used for all the WSO2 PMT communications.
+ *
+ * @since 1.0.0
+ */
+public class PmtApiCaller {
```

Comment:
As same as GithubApiCaller, we can make PmtApiCaller, a static one too, reason -> as we do not see any use of instance variables, WDYT?